### PR TITLE
Replacements for deprecated getCfg()

### PR DESCRIPTION
**Pull request by frankmayer**
Replaces deprecated getCfg() with get()

### CODE REVIEW

File: tests/unit/suites/libraries/cms/application/JApplicationCmsTest.php [hunk 1]
```diff
@@ -256,6 +256,23 @@ public function testExecuteWithDocument()
 	}
 
 	/**
+	 * Tests the JApplicationCms::get method.
+	 *
+	 * @return  void
+	 *
+	 * @since   3.2
```
Comment (andrepereiradasilva) [inline tests/unit/suites/libraries/cms/application/JApplicationCmsTest.php]: Use `__DEPLOY_VERSION__`

Comment (frankmayer) [inline tests/unit/suites/libraries/cms/application/JApplicationCmsTest.php]: Thanks!!! :)

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 6ba649939470dcbfaff23a1d481d4181f8886b7e<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13161">issues.joomla.org/tracker/joomla-cms/13161</a>.</sub>

**Comment by frankmayer** [comment]
Should I close this one,  and make a new PR against 4.0? Opinions?

**Comment by andrepereiradasilva** [comment]
why?

**Comment by frankmayer** [comment]
Forget it, thought it was deprecating for 4.0 also. It's just that I have so many PRs and branches lying around... want to get at least the smaller ones get merged :smile:

**Comment by frankmayer** [comment]
@Bakual Thanks :+1:

**Comment by frankmayer** [comment]
@bakual It is a pleasure :+1: 
... and thanks for the prompt merging just now. 
Now, to be able to continue doing those mass things, I would need to get as many as possible of my other PR's closed, too, in order to avoid constant conflict resolution. It would be great if we could get those things merged as soon as possible.

**Pull request merged**